routing_model/src/scenario_tests.rs

```diff
@@ -893,10 +890,7 @@ mod dst_tests {
     fn test_parsec_offline_then_local_event_back_online() {
```

Comment:
I am not sure this test is necessary, is there any difference with what is tested by test_local_events_offline_online_again_for_different_nodes?
The one bellow that seem useful though.

Reply:
In the second case you're detecting back online a node that was consensus-ed away, maybe that doesn't matter? To be on the safe side I think we should keep both

Reply:
There are many thing that could be tested, so we have to make choices. I would not think keeping an extra test is necessarily the safe option since it adds maintainability cost. (The main good thing about TDD is that it guides you to only add test you need. (I'm not really a practitioner). In this case what code would you have to write to make this test pass that did not exist for the previous one?)

The local event should always be voted in Parsec, so trying that sepcific circumstance does not seem to add much, but I'll let you decide :-)

Reply:
Fair point, ok I'll remove!